### PR TITLE
Translation to Russian

### PR DESCRIPTION
**Pull request by pdumin**
Chapter0/1, chapter1/1-3 finished

Related to #75

### CODE REVIEW

File: .github/workflows/build_documentation.yml [hunk 1]
```diff
@@ -14,6 +14,6 @@ jobs:
       package: course
       path_to_docs: course/chapters/
       additional_args: --not_python_module
-      languages: en es fr he pt th tr
+      languages: en es fr he pt ru th tr
```
Comment (lewtun) [inline .github/workflows/build_documentation.yml:17]: FYI I've added Russian to this list so we can test the docs build in the CI :)

**Review by lewtun — APPROVED** [review]
Thank you for this great contribution 🤗 ! There's just a small change needed on the table of contents, but otherwise this looks good to me (I did a quick translation to English)

File: chapters/ru/_toctree.yml [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+
+- title: 1. Transformer models
+  sections:
+  - local: chapter1/1
+    title: Введение
+  - local: chapter1/2
+    title: Обработка естесственного языка
+  - local: chapter1/3
+    title: Трансформеры, на что они способны?
+  # - local: chapter1/4
```
[the remark this answers is not in the document]
Comment (pdumin) [reply chapters/ru/_toctree.yml]: Removed what you asked for.

**Comment by lewtun** [comment]
The docs build looks good: https://moon-ci-docs.huggingface.co/course/pr_88/ru/chapter1/1

**Comment by lewtun** [comment]
Thank you for iterating on this! Merging now and will deploy to the website today :)

**Comment by blademoon** [comment]
Good afternoon. Can you tell me for what reason the translated subtitles of the first part of the course from YouTube were removed?

**Pull request merged**